release-notes/opensearch-sql.release-notes-1.3.0.0.md

```diff
@@ -0,0 +1,41 @@
+### Version 1.2.4.0 Release Notes
+
+Compatible with OpenSearch and OpenSearch Dashboards Version 1.3.0
+
+### Enhancements
+* PPL integration with AD and ml-commons ([#468](https://github.com/opensearch-project/sql/pull/468))
+* Support ISO 8601 Format in Date Format. ([#460](https://github.com/opensearch-project/sql/pull/460))
+* Treating ExpressionEvaluationException as client Error. ([#459](https://github.com/opensearch-project/sql/pull/459))
+* Add Certificate Validation option ([#449](https://github.com/opensearch-project/sql/pull/449))
+* Span expression should always be first in by list if exist ([#437](https://github.com/opensearch-project/sql/pull/437))
+* Support multiple indices in PPL and SQL ([#408](https://github.com/opensearch-project/sql/pull/408))
+* Support combination of group field and span in stats command ([#417](https://github.com/opensearch-project/sql/pull/417))
+* Support In clause in SQL and PPL ([#420](https://github.com/opensearch-project/sql/pull/420))
+* Add cast function to PPL ([#433](https://github.com/opensearch-project/sql/pull/433))
+* Add parse command to PPL ([#411](https://github.com/opensearch-project/sql/pull/411))
+* [Enhancement] optimize sort rewrite logic ([#434](https://github.com/opensearch-project/sql/pull/434))
+
+### Bug Fixes
+* Bug Fix, disable html escape when formatting response ([#412](https://github.com/opensearch-project/sql/pull/412))
+* Parse none type field as null instead of throw exception ([#406](https://github.com/opensearch-project/sql/pull/406))
+
+### Documentation
+* Add parse docs to PPL commands index ([#486](https://github.com/opensearch-project/sql/pull/486))
+* Add limitation section in PPL docs ([#456](https://github.com/opensearch-project/sql/pull/456))
+* Add how to setup aws credentials for ODBC Tableau ([#394](https://github.com/opensearch-project/sql/pull/394))
+
+### Maintenance
+* Add JDK 8 to CI Matrix  ([#483](https://github.com/opensearch-project/sql/pull/483))
+* Add CI Matrix for JDK 11 and 14 ([#451](https://github.com/opensearch-project/sql/pull/451))
+* Update backport and add auto-delete workflows ([#446](https://github.com/opensearch-project/sql/pull/446))
+* Add auto backport functionality for SQL ([#445](https://github.com/opensearch-project/sql/pull/445))
```

Comment:
similar comment as in the observability one: CI/workflows/tests are in infrastructure, third-party library version bumps are usually to fix CVEs so they are bug fixes. Also 479 is a bug fix, others look good to me